src/apps/schema/src/app/views/SchemaEdit/FieldAdd/FieldAdd.less

```diff
@@ -15,25 +15,4 @@
       font-size: 32px;
     }
   }
-  main {
```

Comment:
This deletion seems like it have a large effect on styling with nothing to replace it.